src/app/campaign/flight/flight.component.ts

```diff
@@ -81,4 +96,12 @@ export class FlightComponent implements OnInit {
   updateFlightForm({ startAt, endAt, ...restOfFlight }: Flight) {
     this.flightForm.reset({ startAt: new Date(startAt), endAt: new Date(endAt), ...restOfFlight }, { emitEvent: false });
   }
+
```

Comment:
🙃  _smiles at reverse naming_
Typically, Angular style would have you name the functions similarly but prefixed with "on". So `onFlightDuplicate()` would call `this.flightDuplicate.emit(this.flight)` or `onDuplicateFlight() with `duplicateFlight.emit()`. The convention is to use "on" prefix with the function and without the prefix in the template/Output.

Reply:
Good call! I knew the reverse naming was silly while I was doing it, couldn't come up with anything better at the time.